src/RileyMaze/toolbox.js

```diff
@@ -92,6 +81,8 @@ var RMZToolbox = GObject.registerClass(class RMZToolbox extends Toolbox {
         }
 
         this._instructTopic.unbindGlobalModel();
+        this._unitTopic.unbindGlobalModel();
```

Comment:
Looks like adding these calls is fixing a separate bug? If so, put it in a separate commit.